packages/frameworks/src/frameworks.ts

```diff
@@ -1932,6 +1934,7 @@ export const frameworks = [
     ],
   },
   {
+    // TODO: fix detected as "vite"
```

Comment:
Should "Hydrogen" be moved above "Vite"?

Reply:
@TooTallNate @leerob what do you think?

Reply:
I think the answer is yes.

Also cc @TooTallNate

Reply:
I think yes.

Reply:
done: [`f880f35` (#9197)](https://github.com/vercel/vercel/pull/9197/commits/f880f355022a32c845fc589a188c5399d2a63d8b)